src/app-gocardless/banks/bankinter_bkbkesmm.js

```diff
@@ -9,19 +8,7 @@ export default {
 
   institutionIds: ['BANKINTER_BKBKESMM'],
 
-  accessValidForDays: 90,
-
-  normalizeAccount(account) {
-    return {
-      account_id: account.id,
-      institution: account.institution,
-      mask: account.iban.slice(-4),
-      iban: account.iban,
-      name: [account.name, printIban(account)].join(' '),
-      official_name: account.product,
-      type: 'checking',
-    };
-  },
+  accessValidForDays: 180,
```

Comment:
Changing this for the existing bank adapters is quite risky unless you are able to test them all. There might be a reason why someone has hardcoded it to 90 here and by changing it to 180 - we risk breaking this integration.

Reply:
I checked all of them against the current info returned by GoCardless, I have a feeling most of these are hardcoded to 90 because the template was copied directly in the past

This is in preparation of a bigger change where we can use the max available instead of a hardcoded 90